### PR TITLE
Followup to [DOC-5841]

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-5841]

## Previews

[cockroachcloud/create-your-cluster.md](https://deploy-preview-15480--cockroachdb-docs.netlify.app/docs/cockroachcloud/create-your-cluster.html)
[v22.2/recommended-production-settings.md](https://deploy-preview-15480--cockroachdb-docs.netlify.app/docs/v22.2/recommended-production-settings.html)
[v22.1/recommended-production-settings.md](https://deploy-preview-15480--cockroachdb-docs.netlify.app/docs/v22.1/recommended-production-settings.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

File: cockroachcloud/create-your-cluster.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ Currently, you can add a maximum of 150 nodes to your cluster. For larger config
 
 ## Step 5. Select the hardware per node
 
+{% capture cap_per_vcpu %}{% include_cached {{ page.version.version }}/prod-deployment/provision-storage.md %}{% endcapture %}
```
Comment (mdlinville) [inline cockroachcloud/create-your-cluster.md:74]:
```suggestion
{% capture cap_per_vcpu %}{% include_cached ../v22.1/prod-deployment/provision-storage.md %}{% endcapture %}
```

**Comment by mdlinville** [comment]
> I can't actually see the previews? Get some `**Not Found - Request ID: 01GGABQA5044MEK4E04QDVBZP5**` error message.
> 
> Also couldn't find what the value is set to on the macro for `cap_per_vcpu`, but the feedback we got from the field is that the way it's worded is confusing, and something more like `up to 320 GiB/vCPU` is easier to understand rather than a range.

A build error prevented the preview from being built. Sorry about that.

I've updated the wording in the include files to no longer use a range (so you can also see them in this PR).

File: v22.1/recommended-production-settings.md [hunk 2]
```diff
@@ -64,12 +64,39 @@ Once you have [sized your cluster](#sizing), derive the amount of RAM, storage c
 
 This hardware guidance is meant to be platform agnostic and can apply to bare-metal, containerized, and orchestrated deployments. Also see our [cloud-specific](#cloud-specific-recommendations) recommendations.
 
-| Value | Recommendation | Reference
-|-------|----------------|----------
-| RAM per vCPU | 4 GiB | [Memory](#memory)
-| Capacity per vCPU | 150 GiB | [Storage](#storage)
-| IOPS per vCPU | 500 | [Disk I/O](#disk-i-o)
-| MB/s per vCPU | 30 | [Disk I/O](#disk-i-o)
+{% capture cap_per_vcpu %}{% include_cached {{ page.version.version }}/prod-deployment/provision-storage.md %}{% endcapture %}
```
Comment (mdlinville) [inline v22.1/recommended-production-settings.md:67]:
```suggestion
{% capture cap_per_vcpu %}{% include_cached v22.1/prod-deployment/provision-storage.md %}{% endcapture %}
```

File: v22.2/recommended-production-settings.md [hunk 3]
```diff
@@ -64,12 +64,39 @@ Once you have [sized your cluster](#sizing), derive the amount of RAM, storage c
 
 This hardware guidance is meant to be platform agnostic and can apply to bare-metal, containerized, and orchestrated deployments. Also see our [cloud-specific](#cloud-specific-recommendations) recommendations.
 
-| Value | Recommendation | Reference
-|-------|----------------|----------
-| RAM per vCPU | 4 GiB | [Memory](#memory)
-| Capacity per vCPU | 150 GiB | [Storage](#storage)
-| IOPS per vCPU | 500 | [Disk I/O](#disk-i-o)
-| MB/s per vCPU | 30 | [Disk I/O](#disk-i-o)
+{% capture cap_per_vcpu %}{% include_cached {{ page.version.version }}/prod-deployment/provision-storage.md %}{% endcapture %}
```
Comment (mdlinville) [inline v22.2/recommended-production-settings.md:67]:
```suggestion
{% capture cap_per_vcpu %}{% include_cached v22.1}/prod-deployment/provision-storage.md %}{% endcapture %}
```

File: cockroachcloud/create-your-cluster.md [hunk 4]
```diff
@@ -71,6 +71,8 @@ Currently, you can add a maximum of 150 nodes to your cluster. For larger config
 
 ## Step 5. Select the hardware per node
 
+{% capture cap_per_vcpu %}{% include_cached ../v22.1/prod-deployment/provision-storage.md %}{% endcapture %}
```
Comment (mdlinville) [inline cockroachcloud/create-your-cluster.md:74]:
```suggestion
{% capture cap_per_vcpu %}{% include_cached v22.1/prod-deployment/provision-storage.md %}{% endcapture %}
```

**Review by mwang1026 — APPROVED** [review]
Preview still doesn't work. But based on my cursory reading of the changelog I think it's OK to push this (YOLO!) and we can do anymore tweaks if needed. Thanks for gritting through the netlify issues.

**Comment by mdlinville** [comment]
> cockroachcloud/create-your-cluster.md

I think you were just a couple minutes too early. The previews work for me now. Thanks for your patience!

**Pull request merged**